doc/api/http2.md

```diff
@@ -1474,6 +1474,12 @@ added: v8.4.0
     used to determine the padding. See [Using options.selectPadding][].
   * `settings` {[Settings Object][]} The initial settings to send to the
     remote peer upon connection.
+  * `ServerRequest` {Http2ServerRequest} Specifies the ServerRequest class to
```

Comment:
Nit: I would use `http2.Http2ServerRequest` for consistency.

Reply:
@hekike sorry I wasn't clear. I didn't mean to change the option name but it's type from `{Http2ServerRequest}` to `{http2.Http2ServerRequest}` because this is what it is used in the rest of the file.